### PR TITLE
Added Help to each right-click menu and also to the script window

### LINKED ISSUE #7067: Adding Help to each right-click menu
When teaching with R-Instat the tutorials and our courses introduce the right-click menus early on.  
These menus don't yet all have their own Help option.

This issue proposes a Help option be the bottom option in each right-click menu, for the data grid, and also have a separator line above it.  
Currently there is a Help option on right-click in the Script window, Log window.  
In the data window there is no help option on the right-click menus.  I suggest it be added and should go to the right click option - help context 146.  This is for all  3 right-click menus.   

In addition, the script window now has 3 buttons to help its use.  Could there be another Help button as well.  It will be the same location as the right-click on the window, i.,e go to context number 542.

In the old output window it is also there - at least in Version 7.3.1, but without the line.  In the new output window there is no right-click, but there are tabs at the top.  @ChrisMarsh82 Could help be added to these?  The help context is 540.
Also, @ChrisMarsh82 I think you mentioned you don't like the right-click menus?  So you may have problems with this issue.  Good to get your reaction, before @Wycklife possibly adds the help buttons to the data window right-click menus.

### PR DESCRIPTION
**Pull request by Wycklife**
Fixes #7067

@africanmathsinitiative/developers This is ready for review

### CODE REVIEW

**Comment by rdstern** [comment]
@Wycklife I like the fact the help on (even right-click) on the script window, goes to the script window
But I think the same on the log window also goes to the script window.  So that should change.
The main right-click menu is also available on the left in the column metadata window.  Add the help there, just as in the data window, and it goes to the same place.
There is also a right-click in the data selector in each dialogue.  You could add Help there as well.  It could go to the `Controls on Each Dialogue` in the `Menus and Dialogues` section

**Comment by Wycklife** [comment]
@rdstern You mean help on Log window goes to the script window? I thought it should go to the Log window?

**Comment by rdstern** [comment]
@Wycklife it currently goes to the script window.  I also think it should go to the log window

**Comment by Wycklife** [comment]
@rdstern Let me check a gain, thanks.

**Comment by Wycklife** [comment]
@rdstern I have made the changes you requested. Could you please test and let me know. Thank you.

**Review by rdstern — APPROVED** [review]
@Wycklife this all looks good now.  Great.  With a check from @lloyddewit this could be merged.

File: instat/ucrColumnMetadata.vb [hunk 1]
```diff
@@ -435,4 +435,12 @@ Public Class ucrColumnMetadata
...
         EndWait()
     End Sub
+
+    Private Sub mnuHelp1_Click(sender As Object, e As EventArgs) Handles mnuHelp1.Click
+        Help.ShowHelp(Me, frmMain.strStaticPath & "\" & frmMain.strHelpFilePath, HelpNavigator.TopicId, "543")
+    End Sub
+
+    Private Sub mnuHelp2_Click(sender As Object, e As EventArgs) Handles mnuHelp2.Click
+        Help.ShowHelp(Me, frmMain.strStaticPath & "\" & frmMain.strHelpFilePath, HelpNavigator.TopicId, "543")
+    End Sub
```
Comment (lloyddewit) [inline instat/ucrColumnMetadata.vb:445]: Combine these into a single sub? e.g. something like:
```suggestion
    Private Sub mnuHelp_Click(sender As Object, e As EventArgs) Handles mnuHelp1.Click, mnuHelp2.Click
        Help.ShowHelp(Me, frmMain.strStaticPath & "\" & frmMain.strHelpFilePath, HelpNavigator.TopicId, "543")
    End Sub
```

File: instat/ucrDataView.vb [hunk 2]
```diff
@@ -858,4 +858,20 @@ Public Class ucrDataView
...
+        Help.ShowHelp(frmMain, frmMain.strStaticPath & "/" & frmMain.strHelpFilePath, HelpNavigator.TopicId, "146")
+    End Sub
+
+    Private Sub mnuHelp2_Click(sender As Object, e As EventArgs) Handles mnuHelp2.Click
+        Help.ShowHelp(frmMain, frmMain.strStaticPath & "/" & frmMain.strHelpFilePath, HelpNavigator.TopicId, "146")
+    End Sub
+
+    Private Sub mnuHelp3_Click(sender As Object, e As EventArgs) Handles mnuHelp3.Click
+        Help.ShowHelp(frmMain, frmMain.strStaticPath & "/" & frmMain.strHelpFilePath, HelpNavigator.TopicId, "146")
+    End Sub
```
Comment (lloyddewit) [inline instat/ucrDataView.vb:876]: Combine these into a single sub?

File: instat/ucrScript.vb [hunk 3]
```diff
@@ -275,4 +275,7 @@ Public Class ucrScript
         Help.ShowHelp(Me, frmMain.strStaticPath & "\" & frmMain.strHelpFilePath, HelpNavigator.TopicId, "542")
     End Sub
 
+    Private Sub cmdHelp_Click(sender As Object, e As EventArgs) Handles cmdHelp.Click
+        Help.ShowHelp(Me, frmMain.strStaticPath & "\" & frmMain.strHelpFilePath, HelpNavigator.TopicId, "542")
+    End Sub
```
Comment (lloyddewit) [inline instat/ucrScript.vb:280]: Merge into sub above?

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@Wycklife thanks for the changes, just a couple of questions

File: instat/ucrColumnMetadata.vb (hunk 1 above)
Comment (Wycklife) [reply instat/ucrColumnMetadata.vb]: Fixed

File: instat/ucrDataView.vb (hunk 2 above)
Comment (Wycklife) [reply instat/ucrDataView.vb]: Fixed

File: instat/ucrScript.vb (hunk 3 above)
Comment (Wycklife) [reply instat/ucrScript.vb]: Fixed.

**Review by lloyddewit — APPROVED** [review]
@Wycklife thanks for the changes, happy to approve

**Comment by lloyddewit** [comment]
@rdstern The changes should not have affected the functionality but it would be good to check. If you're able to test/approve again, then we can merge, thanks

**Review by rdstern — APPROVED** [review]
Still good

**Pull request merged**